content/docs/other-guides/job-scheduling.md

```diff
@@ -4,16 +4,16 @@ description = "How to schedule a job with gang-scheduling"
 weight = 30
 +++
 
-This guide describes how to use [volcano scheduler](https://github.com/volcano-sh/scheduler) to support gang-scheduling in 
+This guide describes how to use [volcano scheduler](https://github.com/volcano-sh/volcano) to support gang-scheduling in 
 Kubeflow, to allow jobs to run multiple pods at the same time.
 
 ## Running jobs with gang-scheduling
 To use gang-scheduling, you have to install volcano scheduler in your cluster first as a secondary scheduler of Kubernetes and configure operator to enable gang-scheduling. 
 
-* Volcano's scheduler repo is [here](https://github.com/volcano-sh/scheduler)  and check how to install it [here](https://github.com/volcano-sh/volcano).
+* Volcano's repo is [here](https://github.com/volcano-sh/volcano)  and check how to install it [here](https://github.com/volcano-sh/volcano).
```

Comment:
Since the two links are now the same, please change the wording to this:

```
* Follow the [instructions in the Volcano repo](https://github.com/volcano-sh/volcano) to install Volcano.
```

Reply:
OK, SGTM